vtrace/platforms/gdbstub.py

```diff
@@ -661,7 +684,12 @@ def _connectSocket(self):
         self._gdb_sock.connect((self._gdb_host, self._gdb_port))
         self._gdb_sock.settimeout(None)
 
-    
+
+    def __del__(self):
+        if self._gdb_sock is not None:
+            self._gdb_sock.close()
+            self._gdb_sock = None
+
```

Comment:
ummm, didn't we just define this above?  and they're different.  what exactly do you want to do here, @sprout42 ? :p

Reply:
oops, this is a duplicate, I'll remove it.